### PR TITLE
Updating the README to include a banner to the new repo

### PR DESCRIPTION
**Pull request by calebk1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,13 @@
 description: Welcome to the home of Adobe Experience Platform Mobile SDK documentation.
 ---
 
+{% hint style="danger" %}
```
Comment (emdobrin) [inline README.md:5]: should we mark this as info?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+## The Mobile SDK documentation has moved!
+
+The Adobe Experience Platform Mobile SDK documentation has now moved. The latest version of the documentation can be found at the [Adobe Developer site](https://developer.adobe.com/client-sdks/documentation/) at https://developer.adobe.com/client-sdks/documentation/.
```
Comment (emdobrin) [inline README.md:8]: I suggest we only keep one link for people to click on, for example:
The Adobe Experience Platform Mobile SDK documentation has now moved. The latest version of the documentation can be found at the Adobe Developer site at https://developer.adobe.com/client-sdks/documentation.

Comment (swarna04) [reply README.md:8]: Yes the double links are unnecessary!

Comment (swarna04) [reply README.md:8]: @calebk1 Can we also embed an item in the header that links to the new doc location as header is visible of every page.

Comment (calebk1) [reply README.md:8]: Yeah, the double link was more I wanted to be REALLY SUPER explicit 😂 

But yeah, I can make it just one link.

Comment (calebk1) [reply README.md:8]: @swarna04 I don't think I can easily do that unless I just go into every doc and paste it on... But I'm not sure if that's the best scenario to do.

File: README.md (hunk 1 above)
Comment (calebk1) [reply README.md:5]: Got it! I had it as Danger purely because I thought that that would make it stand out more.

**Comment by sbenedicadb** [comment]
@swarna04 @emdobrin we good to merge this in?

**Pull request merged**